lib/rust/ast/src/generation.rs

```diff
@@ -0,0 +1,324 @@
+//! This module exports scala ast generator.
```

Comment:
Please run `cargo check` on all of your files and fix the errors it brings up. CI Is showing a bunch of them.